runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -174,15 +174,18 @@ public void keyPressed(KeyEvent e)
 					.getData(DataFlavor.stringFlavor)
 					.toString()
 					.trim();
+				final int maxEmailLength = 254, maxPasswordLength = 20;
```

Comment:
constant in java = private/public static final, this is just named variable

Reply:
Alright, think I got that right this time in the latest commit.